deepmd/pt/utils/utils.py

```diff
@@ -11,17 +11,164 @@
 import torch.nn.functional as F
 
 from deepmd.dpmodel.common import PRECISION_DICT as NP_PRECISION_DICT
+from deepmd.pt.utils import (
+    env,
+)
 
 from .env import (
     DEVICE,
 )
 from .env import PRECISION_DICT as PT_PRECISION_DICT
 
 
+@torch.jit.script
+def silut_forward(
```

Comment:
I am unsure if the decorator increases the import timing. If so, we can compile the function in the runtime.

https://github.com/deepmodeling/deepmd-kit/blob/70bc6d89d057d348558b87865ff2f225cae677aa/deepmd/pt/utils/neighbor_stat.py#L138